pkg/api/v1alpha1/cloudstackdatacenterconfig_test.go

```diff
@@ -39,14 +39,24 @@ func TestGetCloudStackDatacenterConfig(t *testing.T) {
 				},
 				Spec: CloudStackDatacenterConfigSpec{
 					Domain: "domain1",
-					Zone: CloudStackResourceRef{
-						Value: "zone1",
-						Type:  Name,
-					},
 					Account: "admin",
-					Network: CloudStackResourceRef{
-						Value: "net1",
-						Type:  Name,
+					Zones: []Zone{
+						{
+							ZoneRef: CloudStackResourceRef{
```

Comment:
spacing is off here. Have you run `make lint`?

Reply:
run gofumpt after make lint, should be good now.